src/creditCardList.js

```diff
@@ -7,7 +7,7 @@ const CREDIT_CARD_LIST = [
     regexpFull: '^3(?:0[0-5]|[68][0-9])[0-9]{11}$'
   }, {
     name: 'Discover',
-    regexpFull: '^6(?:011|5[0-9]{2})[0-9]{12}$'
+    regexpFull: '^6(?:011|5[0-9]{2}|4[4-9][0-9]{1}|(22(12[6-9]|1[3-9][0-9]|[2-8][0-9]{2}|9[01][0-9]|92[0-5]$)[0-9]{10}$))[0-9]{12}$'
```

Comment:
I should you doing this validation by validating the number in a range of numbers, it would be more readable, but it's ok in a regex.